core/unit_test/cuda/TestCuda_InterOp_Streams.cpp

```diff
@@ -193,7 +193,7 @@ TEST(cuda, raw_cuda_streams) {
   CUDA_SAFE_CALL(cudaDeviceSynchronize());
   cudaStreamDestroy(stream);
 
-  int* h_p = new int[100];
```

Comment:
ah does that resolve a leak?

Reply:
Yes, we never freed the memory allocated here.